index.html

```diff
@@ -111,7 +111,7 @@ <h1 class="heading-big text-uppercase text-purple text-center">Features</h1>
       <h1 class="heading-big text-uppercase text-purple text-center">Ready to roll?</h1>
       <div class="action-btns text-center">
         <a href="https://envoyproxy.github.io/envoy/index.html" class="btn btn-primary">Get Started</a>
-        <a href="download" class="btn btn-secondary">Download</a>
+        <a href="/download" class="btn btn-secondary">Download</a>
       </div>
```

Comment:
This is a good catch. I think we should point to the official docs for now though, but open to discussion.